File: 242-valid-anagram/valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if(s.size() != t.size()){
            return false;
        };
        vector<int> countS(26,0);
        vector<int> countT(26,0);
        for(int i=0; i<s.size(); i++){
            countS[s[i]-'a']++;
            countT[t[i]-'a']++;
        };
        return countS == countT;
    }
};

File: 242-valid-anagram/valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        if ( len(s) != len(t)):
            return False
        frq = [0] * 26
        for ch in s:
            frq[ord(ch) -ord('a')] +=1
        for ch in t:
            frq[ord(ch) - ord('a')] -=1
        for c in frq:
            if(c !=0):
                return False
        return True
        
        